src/commands/duel.ts

```diff
@@ -233,6 +226,52 @@ export class Duel {
     }
   }
 
+  @Slash('duelstreaks', { description: 'Show the overall duel statistics' })
+  private async streaks(interaction: CommandInteraction) {
+    const streakStats = ['winStreakMax', 'lossStreakMax', 'draws', 'losses', 'wins'] as const
+    const statFormatter = async (statName: (typeof streakStats[number]), emptyText: string): Promise<string> => {
+      let stats = await this.client.$queryRawUnsafe<Duels[]>(`SELECT * FROM Duels WHERE ${statName}=(SELECT MAX(${statName}) FROM Duels) AND ${statName} > 0`)
+      if(stats.length == 0) {
+        return emptyText
+      }
+      let extraMessage = ''
+      if(stats.length > 2) {
+        // If more than 2 users share the same stat, shuffle the array and only print the first two.
+        // The rest will be listed as n other users
+        extraMessage = `, and ${stats.length - 2} other user${stats.length - 2 > 1 ? 's' : ''}`
+        stats = shuffleArray(stats).slice(0, 2)
+      }
+      const statHavers = await Promise.all(stats.map(async (duel) => {
+        const user = await interaction.client.users.fetch(duel.userId)
```

Comment:
Should reference the nickname first unless none is set then use the username.
```suggestion
        const member = await interaction.guild?.members.fetch(duel.userId)
        return member?.nickname ?? member?.user?.username ?? ''
```